tests/test_tracer.py

```diff
@@ -530,3 +530,18 @@ def test_tracer_url():
     with pytest.raises(ValueError) as e:
         t = ddtrace.Tracer(url='foo://foobar:12')
         assert str(e) == 'Unknown scheme `https` for agent URL'
+
+
+def test_tracer_shutdown():
```

Comment:
is it really worth it?
It's not really testing anything 🤷‍♂

Reply:
🤷‍♂ keeps test coverage up 😆

Reply:
A good summary why coverage isn't a relevant metric :)